test/Verifiers.test.ts

```diff
@@ -144,8 +142,6 @@ contract('DeployVerifier', function ([relayHub, other1, relayWorker, verifierOwn
   })
 
   it('SHOULD fail on Balance Too Low of preRelayCall', async function () {
-    await deployVerifier.acceptToken(token.address, { from: verifierOwner })
```

Comment:
I assume these were redundant and now cause a revert and for that reason they were removed

Reply:
> I assume these were redundant and now cause a revert and for that reason they were removed

you're exactly right :+1: